src/screens/login/Login.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { StatusBar, View } from 'react-native';
+import CheckBox from '@react-native-community/checkbox';
+import * as S from './styled';
+
+import AppButton from '../../components/AppButton';
+import InputField from '../../components/InputField';
+import { routes } from '../../constants/navigation/constants';
```

Comment:
⛏️ Sempre bom manter as importações locais após as importações externas
```suggestion
import AppButton from '../../components/AppButton';
import InputField from '../../components/InputField';
import { routes } from '../../constants/navigation/constants';

import * as S from './styled';
```

Reply:
👍